config/prometheus/collector_alerts.yaml

```
@@ -55,6 +55,36 @@ spec:
       labels:
         service: collector
         severity: Warning
+    - alert: ElasticsearchDeprecation
+      annotations:
+        message: "The OpenShift Elasticsearch Operator is deprecated and is planned to be removed in a future release. Red Hat provides bug fixes and support for this feature during the current release lifecycle, but this feature no longer receives enhancements. As an alternative to using the OpenShift Elasticsearch Operator to manage the default log storage, you can use the Loki Operator."
+        summary: "Detected Elasticsearch as the on cluster storage which is being deprecated in a future release."
+      expr: |
+        sum(min_over_time(container_cpu_usage_seconds_total{container=~'elasticsearch'}[5m])) > 0
+      for: 10m
+      labels:
+        service: storage
+        severity: Warning
+    - alert: FluentdDeprecation
+      annotations:
+        message: "Fluentd is deprecated and is planned to be removed in a future release. Red Hat provides bug fixes and support for this feature during the current release lifecycle, but this feature no longer receives enhancements. As an alternative to Fluentd, you can use Vector instead."
+        summary: "Detected Fluentd as the collector which is being deprecated in a future release."
+      expr: |
+        sum(min_over_time(container_cpu_usage_seconds_total{image=~'.+fluentd.+'}[5m])) > 0
+      for: 10m
+      labels:
+        service: collector
+        severity: Warning
+    - alert: KibanaDeprecation
+      annotations:
+        message: "The Kibana web console is now deprecated and is planned to be removed in a future logging release."
+        summary: "Detected Kibana as the visualization which is being deprecated in a future release"
+      expr: |
+        sum(min_over_time(container_cpu_usage_seconds_total{container=~'kibana'}[5m])) > 0
```

See `elasticsearch` for the longer explanation:

```suggestion
        sum(kube_pod_labels{namespace="openshift-logging", label_component="kibana"}) > 0
```